.github/pull-request-template.md

```diff
@@ -14,6 +14,17 @@ Changes proposed in this pull request:
 -
 -
 
-**Issue link**
 
+**Related issue(s)**
 <!-- If you refer to a particular issue, provide its number. For example, `Resolves #123`, `Fixes #43`, or `See also #33`. -->
+
+**Special notes for your reviewer**
+
+
+**Release note**
```

Comment:
Hmm, so basically release notes are duplicated description what this pull request brings... One of the sections is redundant from my perspective

Reply:
Not necessarily. First off, not all PRs are relevant for release notes. Also, it could be that the description also needs to include some technical details, which are also not relevant for the release notes. The language could also be different...

Reply:
Alright, I agree - but still I'd like to have as little overhead as possible during creating pull requests. It can be automated with the PR labels and the only thing I would need to be careful about is pull request title with description (if someone clicks on a link to PR in Github release).